### PR TITLE
feature: add prettier formatting

### LINKED ISSUE #50: Feature: Add prettier for formatting
We can use the prettier module and set up its config file to autoformat the code on save. This will help to maintain the format of the code even when the contributors make changes to the code.

### PR DESCRIPTION
**Pull request by kailashchoudhary11**
fixes #50

### CODE REVIEW

File: components/Dashboard/History.jsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ const History = () => {
     return <Loader loading={true} />;
   }
   if (error) {
-    return <h2 style={{color: 'wheat'}}>No Music Found</h2>
+    return <h2 style={{ color: "wheat" }}>No Music Found</h2>;
   }
```
Comment (varun-singhh) [inline components/Dashboard/History.jsx:29]: Still I can see single quote is not enabled, Please fix it.

**Review by varun-singhh — CHANGES_REQUESTED** [review]

**Comment by kailashchoudhary11** [comment]
@varun-singhh Done!

**Comment by kailashchoudhary11** [comment]
@varun-singhh  Please merge this so that I can work on linting.

**Comment by varun-singhh** [comment]
@kailashchoudhary11 resolve conflicts

**Comment by kailashchoudhary11** [comment]
@varun-singhh Resolved. Merge it now else there will be conflicts again. 🥲

**Review by varun-singhh — APPROVED** [review]

**Pull request merged**